tests/monitoring/metrics.go

```diff
@@ -142,7 +142,7 @@ func basicVMLifecycle(virtClient kubecli.KubevirtClient) {
 
 func createAndRunVM(virtClient kubecli.KubevirtClient) *v1.VirtualMachine {
 	vmi := libvmifact.NewFedora(libvmi.WithNamespace(testsuite.GetTestNamespace(nil)))
-	vm := libvmi.NewVirtualMachine(vmi, libvmi.WithRunning())
+	vm := libvmi.NewVirtualMachine(vmi, libvmi.WithRunStrategy(v1.RunStrategyAlways))
```

Comment:
Not related to your PR but we should replace this custom function with one that is widely used.

Reply:
Yes, that makes sense to me.